src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatEntries = props.entries.map((entry) => {
```

Comment:
I know that you are using entry.id for the key and still getting an error in learn about the child having a unique key. I would suggest using the index in the map function
```javascript
const chatEntries = props.entries.map((entry, i) => 
...
<li className="chat-log" key={i}
```